lib/graphql/query/literal_input.rb

```diff
@@ -21,7 +21,7 @@ def self.from_arguments(ast_arguments, argument_defns, variables)
             arg_value = coerce(arg_defn.type, ast_arg.value, variables)
           end
           if arg_value.nil?
-            arg_value = arg_defn.default_value
+            arg_value = arg_defn.type.coerce_input(arg_defn.default_value)
```

Comment:
Can we just assign `arg_value = arg_defn.default_value` instead of `arg_value = nil` then avoid this extra condition?